src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -140,6 +140,7 @@ const button = {
   },
   primary: {
```

Comment:
We could infer the `priority` from `Omit<keyof button, 'borderRadius'>`

Reply:
That makes sense, not something I'm going to tackle in this PR though